install.md.hbs

```diff
@@ -273,14 +266,10 @@ this can reuse the `tap-registry` secret created in
 > **Note:** When you install Tanzu Application Platform, it is bootstrapped with
 > a set of dependencies (buildpacks and stacks) for application builds.
 > For more information about buildpacks, see the [VMware Tanzu Buildpacks Documentation](https://docs.vmware.com/en/VMware-Tanzu-Buildpacks/index.html).
-> You can find the buildpack and stack artifacts installed with Tanzu Application Platform
-> in the descriptor file on [Tanzu Network](https://network.pivotal.io/products/tbs-dependencies).
-> The current installed version of the descriptor is
-> [100.0.293](https://network.pivotal.io/products/tbs-dependencies#/releases/1086670). Sometimes the dependencies
-> require updates. You can use a
-> [manual process in a CI/CD context](tanzu-build-service/tbs-about.html#dependencies-manual), or
-> an [automatic update process](tanzu-build-service/tbs-about.html#auto-updates)
-> in the background by Tanzu Application Platform.
+> Sometimes the dependencies require updates. You can use a [manual process in a CI/CD context](tanzu-build-service/tbs-about.html#dependencies-manual), or
+> an experimental [automatic update process](tanzu-build-service/tbs-about.html#auto-updates) in the background by Tanzu Application Platform.
+> By default, a small-footprint (lite) set of dependencies is installed in TAP which does not contain all Buildpacks and Stacks.
+> To install the larger-footprint (full) set of dependencies which contains all buildpacks and stacks, see [Installing the Full Dependencies](tanzu-build-service/tbs-about.html#auto-updates).
```

Comment:
I think we need to let people know here what additional functionality they get out the full set.  Web server buildpack, airgapped support, PHP.

Reply:
let's do it wholesale in https://github.com/pivotal/build-service/issues/766